### ISSUE TITLE
[Switch] ariaLabelledBy not working with label

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-react#1584]
When setting the ariaLabelledBy props of the Switch together with a label, WAVE detects that a form label is present, but it is not correctly associated with a form control. In addition, when the component is focused the screen reader does not read it.

https://stackblitz.com/edit/react-4gs3yx-rrpjhl?file=app%2Fmain.jsx

Ticket ID: 1609995

**Comment by iadnonov** [comment]
In the above stackblitz demo, the `<label />` element do not nave set `for` attribute which to be connected with the `<input />` id.  
The following example showcases all the provided options for setting the label to Kendo React Switch component: https://stackblitz.com/edit/react-gjjnuw-cqkqks?file=app%2Fmain.tsx
